src/instructlab/configuration.py

```diff
@@ -1617,14 +1614,8 @@ def map_train_to_library(ctx, params):
     if params["pipeline"] == "full":
         train_args.disable_flash_attn = True
 
-    student_model_arch = get_model_arch(pathlib.Path(params["model_path"]))
     if ctx.obj.config.general.use_legacy_tmpl:
         train_args.use_legacy_tmpl = True
-    else:
```

Comment:
does this change mean that legacy tokenizer config won't be detected anymore?

Reply:
@booxter We don't need it anymore

Reply:
We don't need the **change** or we don't need **support for legacy** auto-detection? And is this change related to transformers cap removal / training dependency update?